### ISSUE TITLE
Auxiliary/Backup Water Control

### ISSUE THREAD

**Issue by MikaylaFischler** [MikaylaFischler/cc-mek-scada#480]
Support enabling water to either the reactor or the boiler when water levels drop. This can occur on burn rate increases and during initial start. This would activate much earlier than emergency coolant, with emergency coolant being the last line of defense against a meltdown due to loss of coolant.

This would only be implemented for water, auxiliary sodium would not solve the boiler water level low problem. It would also be very challenging to fit into the flow monitor display.

Update: Additionally, auxiliary sodium would end up filling the cooling loop with sodium, since it only gets lost on overfilling. Water should be all that is needed in both water cooled and sodium cooled systems.

- [x] Logic to enable and disable auxiliary coolant
- [x] Supervisor configuration option for auxiliary coolant
- [x] Coordinator updates to render auxiliary coolant valves

**Comment by Sharparam** [comment]
Supplying extra sodium is needed in some way though during ramp up, because it will deplete until empty while a sodium reactor gets up to "speed" (or rather the turbines), unless external sodium is added.

**Comment by MikaylaFischler** [comment]
I disagree with "needed" as that should not be the case. The only cases I've seen are the following, so please elaborate if these don't apply to your situation.

1. **Not Enough Water**: The boiler runs out of water while ramping, so it stops outputting cooled coolant. The fix I observed is supplying extra water to the boiler. This aligns with what some other users have done. Extra sodium would technically work, but that's because dumping infinite sodium into a reactor is an effective cooling method. Doing so will just end up fully filling all the tanks (and pipes if applicable) with sodium and superheated sodium for all future use.
2. **Tanks Too Small**: To run at higher burn rates, there needs to be a significant amount of sodium sitting in the boiler's cooled coolant tank, since a huge amount gets superheated per tick. If the tanks aren't big enough, it can't keep up. When using sodium cooled setups, I manually feed sodium into the reactor until the reactor's tank remains full at its max burn one time and then never again. This requires the boiler's cooled coolant tank to be large enough to store that excess quantity.

The only case where I can imagine extra sodium being necessary is if a turbine filled up its steam tank while ramping (in which case setting it to dumping excess would resolve this), causing the boiler to stop operating, if that's even how the boiler is coded. I haven't verified that yet.
